tools/rln_keystore_generator/external_config.nim

```diff
@@ -51,14 +51,20 @@ type
       defaultValue: "",
       name: "rln-relay-eth-contract-address" }: string
 
-    rlnRelayCredentialsPassword* {.
+    rlnRelayCredPassword* {.
       desc: "Password for encrypting RLN credentials",
       defaultValue: "",
       name: "rln-relay-cred-password" }: string
 
 proc loadConfig*(T: type RlnKeystoreGeneratorConf): Result[T, string] =
   try:
     let conf = RlnKeystoreGeneratorConf.load()
+    if conf.rlnRelayCredPath == "":
+      return err("--rln-relay-cred-path must be set")
+    if conf.rlnRelayEthContractAddress == "":
+      return err("--rln-relay-eth-contract-address must be set")
+    if conf.rlnRelayCredPassword == "":
```

Comment:
Shall we also ask for a minimum password strength? ( e.g. minimum length, a-zA-Z, alphanumeric symbols, ... )

Reply:
can I cover this in a future iteration? tracking issue for it here - https://github.com/waku-org/nwaku/issues/1930

Reply:
Yes of course:) Thanks!

Reply:
unsure if we can do that. is the keystore taken from some spec (eg EIP)? if so, if the EIP does not enforce any password strength, then we can enforce it on top.

eg, not the one we are using but https://eips.ethereum.org/EIPS/eip-2335

> The password is a string of arbitrary unicode characters. The password is first converted to its NFKD representation, then the control codes (specified below) are stripped from the password and finally it is UTF-8 encoded.

there are no requirements beyond that